src/utils.py

```diff
@@ -1152,6 +1163,35 @@ def create_pddl_problem(objects: Collection[Object],
 """
 
 
+def create_video_from_partial_refinements(
+    task: Task, simulator: Callable[[State, Action], State],
+    render: Callable[[State, Task, Optional[Action]], List[Image]],
+    partial_refinements: Sequence[Tuple[Sequence[_GroundNSRT],
+                                        Sequence[_Option]]]
+) -> Video:
+    """Create a video from a list of skeletons and partial refinements."""
+    # Right now, the video is created by finding the longest partial refinement.
+    # We imagine one day also implementing an "all_skeletons" mode that would
```

Comment:
Let's not make a promise like this in the comments...unclear if we'll ever actually do this :P Instead, maybe "One could also implement..."?

Reply:
done